lib/components/narrative/tabbed-itineraries.js

```diff
@@ -130,6 +132,9 @@ class TabButton extends Component {
     // TODO: support non-USD
     const transitFare = (transitFares?.[defaultFareKey] || transitFares.regular)?.transitFare || 0
     const minTotalFare = minTNCFare * 100 + transitFare
+    const formattedStartTime = moment(itinerary.startTime).valueOf() + timezoneOffset
+    const formattedEndTime = moment(itinerary.endTime).valueOf() + timezoneOffset
```

Comment:
Are these instances of `moment().valueOf` UNIX timestamps? If so, you don't need to invoke `moment`. Otherwise, `moment` has methods for time additions that we should use instead. (Also appears in other modified files in this PR)

Reply:
Good point, I'll remove moment calls here and in the other files.

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/898f5695ac99f2f781d768933da78cbbada87161